src/main/java/org/opentripplanner/netex/loader/mapping/DayTypeAssignmentMapper.java

```diff
@@ -21,19 +21,11 @@
 /**
  * Map {@link DayTypeAssignment}s to set of {@link LocalDateTime}.
  * <p/>
- * The mapper is stateful so it is important to follow the following sequence
- * of operations:
+ * The mapper is stateful and uses a 2 step approtch:
  * <p/>
- * At least for each thread:
- * - Create: {@code m = new DayTypeAssignmentMapper(..)} - Setup and inject dependencies
- * - Then for each set of dayTypes in a ServiceJourney (equivalent to a GTFS Calendar Service):
- * <ol>
- *     <li>Init: {@code m.clear() } - Prepare mapper by removing all earlier calculations. The
- *     mapper is not ready to be reused.
- *     <li>Map all: {@code m.map(...) or m.mapAll(...)} - Map all day type assignments using the map methods.
- *     <li>Read result: {@code m.mergeDates()}
- * </ol>
- *
+ * - Create a new mapper
+ * - Add each dayTypeId and list of assignments (multiple calls)
+ * - Merge dates. Take the list of all available days and remove all none available dates.
  * <p/>
```

Comment:
I changed the DayTypeAssignmentMapper so it can be misused any more. The previous version required the caller to invoke `clear()` before using it, if forgotten old results would carry over into the next mappings.